llvm/test/tools/llvm-cas-object-format/csv-stats.test

```diff
@@ -0,0 +1,25 @@
+RUN: rm -rf %t && mkdir -p %t
```

Comment:
Test looks good. You might be able to just write everything in one file unless you are thinking about reusing `.ll` file something else.

Reply:
I think it gives me more flexibility if the .ll file was kept separate for now